guides/concepts/transactions.md

```diff
@@ -17,7 +17,7 @@ Each transaction has the following attributes:
 > #### Sequence number
 > Each transaction has a sequence number. Transactions follow a strict ordering rule when it comes to processing of transactions per account. For the transaction to be valid, the sequence number must match the one stored in the source [account entry](./accounts.md) when the transaction is applied. After the transaction is applied, the source account's stored sequence number is incremented by 1.
 >
-> Note that if several transactions with the same source account make it into the same transaction set, they are ordered and applied according to sequence number. For example, if 3 transactions are submitted and the account is at sequence number 5, the transactions must have sequence numbers 6, 7, and 8.
+> Note that if several transactions with the same source account make it into the same transaction set, they are ordered and applied according to sequence number. For example, if 3 transactions are submitted and the account is at sequence number 5, the transactions must have sequence numbers 5, 6, and 7.
```

Comment:
I'm not sure if this change is correct, can you confirm? @bartekn or @MonsieurNicolas

Reply:
Intention is to keep this consistent with the sentence above, quoting the part that was inconsistent from above:
> For the transaction to be valid, the sequence number must match the one stored in the source

Reply:
Actually the error is in the quoted sentence. A new transaction's sequence number must be equal: current sequence number of the account + 1. So this is correct:
> For example, if 3 transactions are submitted and the account is at sequence number 5, the transactions must have sequence numbers **6, 7, and 8**.

Reply:
ok, thanks. updated to reflect this